batchflow/models/tf/mobilenet.py

```diff
@@ -134,31 +165,44 @@ class MobileNet_v2(TFModel):
         parameters for the initial block (default is 'cna', 32, 3, strides=2)
 
     body : dict
-        layout : list of dict
-            a sequence of block parameters:
+        layout : list of dict each consisting of parameters:
             repeats : int
+                number of repeats of the block
             filters : int
+                number of output filters in the block
             expansion_factor : int
+                multiplier for the number of filters in internal convolutions
             strides : int
-
+                stride for 3x3 convolution
+            kernel_size : int
+                kernel size for depthwise convolution
+            activation : callable, optional
+            se_block : bool or dict
+                whether to include squeeze and excitation block
+                If dict it must contain ratio and activation keys
+                to customize se_block parameters
+            residual : bool
+                whether to make a residual connection
         width_factor : float
             multiplier for the number of channels (default=1)
     """
     @classmethod
     def default_config(cls):
         config = TFModel.default_config()
-        config['common'].update(dict(activation=tf.nn.relu6))
         config['initial_block'].update(dict(layout='cna', filters=32, kernel_size=3, strides=2))
         config['body'].update(dict(width_factor=1, layout=deepcopy(_V2_DEFAULT_BODY)))
         config['head'].update(dict(layout='cnacnV', filters=[1280, 2], kernel_size=1))
-
+        config['common'].update(dict(activation=tf.nn.relu6))
```

Comment:
This will make all activation `relu6` (i.e. in intitial block and head).
If you wanted to have `relu6` in body only, pass it to body

Reply:
in the article and TensorFlow realization it is the same for all layers

Reply:
what about complying with a class docstring - there is no activation for the body?

Reply:
Add it then.
Anyway, by default body is a conv block so it takes all the conv block params

Reply:
No, it doesn't. Anyway. In this case, the same activation goes everywhere.